docs/topics/available-openrewrite-recipes.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following module:
+//
+// * docs/cli-guide-mtr/master.adoc
+
+[id=available-openrewrite-recipes_{context}]
+= Available OpenRewrite recipes
+
+.Available OpenRewrite recipes
+[cols="22%,26%,26%,26%", options="header"]
+|====
+|Migration path
+|Purpose
+|rewrite.configLocation
+|activeRecipes
+
+|Java EE to Jakarta EE
+|Replace import of javax packages with equivalent jakarta packages
```

Comment:
In the Purpose column, use `backticks` around filenames and locations. Likewise `javax` and `jakarta". See https://redhat-documentation.github.io/asciidoc-markup-conventions/.

Reply:
@RichardHoch  Thank you, corrected